.github/workflows/ci.yml

```diff
@@ -48,3 +48,17 @@ jobs:
           LAST_TEAMCITY_BUILD=1007
           export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
           ./script/ci
+
+      - name: Upload to riff-raff
+        uses: guardian/actions-riff-raff@v2
+        with:
+          app: amiable
+          buildNumberOffset: 1007
+          buildNumber: ${{ env.GITHUB_RUN_NUMBER }}
```

Comment:
I don't think we need both `buildNumerOffset` and `buildNumber`. It looks like we can [get away with just `buildNumberOffset`](https://github.com/guardian/security-hq/blob/7012d12bb29faf06bdb10690f58f17cd4ec6200d/.github/workflows/ci-sbt.yml#L54-L55).